.github/workflows/build.yml

```
@@ -183,10 +183,71 @@ jobs:
+    strategy:
+      matrix:
+        python-version: ['3.12']
+    outputs:
+      hash: ${{ steps.get_hash.outputs.hash }}
+      file_name: ${{ steps.get_hash.outputs.file_name }}
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v4
+
+      - name: Setup python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)